src/SUMMARY.md

```diff
@@ -40,10 +40,43 @@
 
 # Day 1: Afternoon
 
+- [Control Flow](control-flow.md)
+  - [Blocks](control-flow/blocks.md)
+  - [if expressions](control-flow/if-expressions.md)
+  - [for expressions](control-flow/for-expressions.md)
+  - [while expressions](control-flow/while-expressions.md)
+  - [break & continue](control-flow/break-continue.md)
+  - [loop expressions](control-flow/loop-expressions.md)
+
 - [Variables](basic-syntax/variables.md)
   - [Type Inference](basic-syntax/type-inference.md)
   - [static & const](basic-syntax/static-and-const.md))
   - [Scopes and Shadowing](basic-syntax/scopes-shadowing.md)
+- [Enums](enums.md)
+  - [Variant Payloads](enums/variant-payloads.md)
+  - [Enum Sizes](enums/sizes.md)
+
+- [Novel Control Flow](advanced-control-flow.md)
+  - [if let expressions](control-flow/if-let-expressions.md)
+  - [while let expressions](control-flow/while-let-expressions.md)
+  - [match expressions](control-flow/match-expressions.md)
+
+- [Pattern Matching](pattern-matching.md)
```

Comment:
I like putting this on day 1 - it's "cool" and powerful and thus motivating, but not as mind-blowing as borrows or move semantics.

Reply:
Me too, I think this will be a nice feature to demonstrate on the first day.